src/Axiom/Set/Map/Dec.agda

```diff
@@ -51,12 +51,12 @@ module Lookupᵐᵈ (sp-∈ : spec-∈ A) where
        helper _ _ | _ , _ , refl | _ , _ , refl | yes _ | yes _
          with refl ← trans (sym eq) eq' = refl
 
-  module _ {V : Type} ⦃ mon : IsCommutativeMonoid' 0ℓ 0ℓ V ⦄ ⦃ _ : DecEq A ⦄ where
-    infixr 6 _∪⁺_
-    open IsCommutativeMonoid' mon
+  open import Algebra
+  module OpUnion {V : Type} (_∙_ : V → V → V) where
+    infixr 6 _∪̇_
 
-    _∪⁺_ : Map A V → Map A V → Map A V
-    _∪⁺_ = unionWith (fold id id _◇_)
+    _∪̇_ : ⦃ DecEq A ⦄ → Map A V → Map A V → Map A V
+    _∪̇_ = unionWith (fold id id _∙_)
```

Comment:
I'm not so sure about this change. It mostly seems to add boilerplate code elsewhere, like for example:
```
_∪⁺_ : ∀ {A} → ⦃ DecEq A ⦄ → A ⇀ ℕ → A ⇀ ℕ → A ⇀ ℕ
_∪⁺_ = OpUnion._∪̇_ _+_
```
If you like `_∪̇_`, why not define `_∪⁺_` with its original type in terms of it here? Same for `aggregate`.

Reply:
I removed it since I don't really need that change.